recipes/antismash/meta.yaml

```diff
@@ -17,45 +18,30 @@ source:
 
 requirements:
   host:
-    - python >=2.7,<3.0a0
+    - python >=3.5
     - pip
-    - icu
   run:
-    - python >=2.7,<3.0a0
-    - argparse
-    - backports.lzma
+    - python >=3.5
     - bcbio-gff
-    - biopython >=1.65
+    - biopython >=1.71
     - blast ==2.2.*
-    - clustalw
-    - cssselect
-    - diamond ==0.8.*
-    - ete2
+    - diamond ==0.9.*
     - fasttree ==2.1.*
     - glimmer ==3.02
     - glimmerhmm ==3.0.*
     - helperlibs
     - hmmer ==3.1b2
     - hmmer2
-    - icu
     - openjdk
-    - libxml2 2.9.*
-    - mafft
     - matplotlib
-    - meme ==4.11.2
-    - muscle
-    - networkx
+    - meme <=4.11.2
```

Comment:
meme <=4.11.2 can't run on python3 environment.

Reply:
antiSMASH doesn't use any Python parts from the meme suite. The `meme` and `fimo` binaries work.

Reply:
Could it be an option to use MEME version 4.11.2 patch 2 -- October 24, 2016 that introduced python 3 support?

Reply:
As long as that has the same output format as 4.11.2, sure. They just changed it somewhere between 4.11.2 and 4.11.4, and we currently don't support the new format.